sdk-extensions/zpages/src/main/java/io/opentelemetry/sdk/extension/zpages/TraceConfigzZPageHandler.java

```diff
@@ -121,7 +121,7 @@ private static void emitChangeTable(PrintStream out) {
         /* rowName= */ "SamplingProbability to",
         /* paramName= */ QUERY_STRING_SAMPLING_PROBABILITY,
         /* inputPlaceHolder= */ "[0.0, 1.0]",
-        /* paramDefaultValue= */ TraceConfig.getDefault().getSampler().getDescription(),
```

Comment:
Pretty sure this was wrong, it would look like `SamplingProbability to ParentBased{AlwaysOnSampler}`